File: problem-20/prb20.c
#include<stdio.h>

int timeEst(int n)
{
    if(n==0)
        printf("\nTime Estimated: 0 minutes");
    else if(n>=1 && n<=2000)
        printf("\nTime Estimated: 25 minutes");
    else if(n>=2001 && n<=4000)
        printf("\nTime Estimated: 35 minutes");
    else if(n>=4001 && n<=7000)
        printf("\nTime Estimated: 45 minutes");
    else if(n>7000)
        printf("\nOVERLOADED");
    else
        printf("\nINVALID INPUT");
}

int main()
{
    int w;
    scanf("%d", &w);
    timeEst(w);
    return 0;
}

File: problem-20/prb20.py
def timeEst(n):
    if(n == 0):
        print("Time Estimated: 0 minutes")
    elif(n >= 1) and (n <= 2000):
        print("Time Estimated: 25 minutes")
    elif(n >= 2001) and (n <= 4000):
        print("Time Estimated: 35 minutes")
    elif(n >= 4001) and (n <= 7000):
        print("Time Estimated: 45 minutes")
    elif(n >= 7001):
        print("OVERLOADED")
    else:
        print("INVALID INPUT")
        
w = int(input())
timeEst(w)
